js/plugins.js

```diff
@@ -29,6 +29,6 @@ module.exports = (ENV) => {
     }),
     new webpack.NamedModulesPlugin(),
     uglifyPlugin,
-    process.env.ANALYZE_BUNDLE && new BundleAnalyzerPlugin()
+    process.env.ANALYZE_BUNDLE && new BundleAnalyzerPlugin(typeof bundleOptions === 'object' ? bundleOptions : {} )
```

Comment:
But it will always be an object, as that is the default.
Unless someone passes in a strange param, in which case I think it's fine to throw an error than silently alter the runtime behaviour.

Reply:
Good point, I'll remove the `typeof check`

Reply:
Actually, I just realised why I did this. Our webpack configs all call pluginJS with two parameters `pluginJS(ENV, PATHS)`.

However the second parameter is currently ignored because nothing is done with it.

That's why I was doing a type check.

In any case, I just figured out you can pass [`{analyzerPort: "auto"}`](https://www.npmjs.com/package/webpack-bundle-analyzer) which will automatically pick an open port, which negates the need to have an extra paremeter.

Reply:
Nice, thanks for explaining. Neat that there was a better solution :)